web/templates/recipient_row.hbs

```diff
@@ -94,9 +94,17 @@
     <div class="message-header-contents">
         <a class="message_label_clickable narrows_by_recipient stream_label tippy-narrow-tooltip"
           href="{{pm_with_url}}"
-          data-tippy-content="{{t 'Go to direct messages with {display_reply_to}' }}">
+          data-tippy-content="{{t 'Go to direct messages with {display_reply_to_for_tooltip}' }}">
         <span class="private_message_header_icon"><i class="zulip-icon zulip-icon-user"></i></span>
-        <span class="private_message_header_name">{{t "You and {display_reply_to}" }}</span>
+        <span class="private_message_header_name">
+            {{#tr}}You and <z-user-names></z-user-names>
+            {{#*inline "z-user-names"}}
+                {{#each recipient_users}}
+                    {{full_name}}{{#if should_add_guest_user_indicator}}&nbsp;<i>({{t 'guest'}})</i>{{/if}}{{#unless @last}},{{/unless}}
```

Comment:
Have done this changes, to avoid using `{{{` but now we have translated guest text inside `#tr` tag.

Reply:
Great, thanks!